web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -636,6 +654,97 @@ export function AppAccessSpecSection({
   );
 }
 
+export function DatabaseAccessSpecSection({
+  value,
+  isProcessing,
+  onChange,
+}: SectionProps<DatabaseAccessSpec>) {
+  return (
+    <>
+      <Box mb={3}>
+        <Text typography="body3" mb={1}>
+          Labels
+        </Text>
+        <LabelsInput
+          disableBtns={isProcessing}
+          labels={value.labels}
+          setLabels={labels => onChange?.({ ...value, labels })}
+        />
+      </Box>
+      <FieldSelectCreatable
+        isMulti
+        label="Names"
+        toolTipContent="List of database names that this role is allowed to connect to"
+        isDisabled={isProcessing}
+        formatCreateLabel={label => `Name: ${label}`}
+        components={{
+          DropdownIndicator: null,
+        }}
+        value={value.names}
+        onChange={names => onChange?.({ ...value, names })}
+      />
+      <FieldSelectCreatable
+        isMulti
+        label="Roles"
+        toolTipContent="List of database roles for automatic user creation"
+        isDisabled={isProcessing}
+        formatCreateLabel={label => `Role: ${label}`}
+        components={{
+          DropdownIndicator: null,
+        }}
+        value={value.roles}
+        onChange={roles => onChange?.({ ...value, roles })}
+      />
+      <FieldSelectCreatable
+        isMulti
+        label="Users"
+        toolTipContent="List of database users that this role is allowed to connect as"
```

Comment:
Would "Database Users" make more sense here, or make it less ambiguous at least? If UX explicitly said use "Users" instead of "Database Users", ignore this comment.